File: 2st/Riyeon/등차수열의 특정한 항만 더하기.java
class Solution {
    public int solution(int a, int d, boolean[] included) {
        int answer = 0, j=a;
        for(int i=0; i < included.length; i++, j+=d){
            if(included[i] == true)
                answer += j;
        }
        return answer;
    }
}

File: 2st/Yeonwoo/등차수열의 특정한 항만 더하기.py
def solution(a, d, included):
    answer = 0
    temp = []
    for i in range(1,len(included)+1):
        su = a+(i-1)*d
        temp.append(su)
    for inx, j in enumerate(included):
        if j==True:
            answer += temp[inx]
    return answer